### DISCUSSION TITLE
Mails bouncing/spam from SL to gmail

### DISCUSSION THREAD

**Discussion by prashantkamdar** [simple-login/app#1449] [Q&A]
Hi,

Lately, any custom alias that I create do not work.
Every time when anyone tries to send email to it, I recieve in my gmail inbox as following:
```An email sent to your alias prashant@custom_domain_here.de from noreply@facebookmail.com was bounced by your mailbox gmail_address_here@gmail.com```

My old aliases work normally, just a problem with the newer ones.
Please let me know what do I need to do to correct this.

**Comment by 4b15ed24f576** [comment]
Hi there,

Could I ask you to contact our support team please?

You can do so by creating a ticket on this page: https://app.simplelogin.io/dashboard/support

Thanks,
